lib/takeaway.rb

```diff
@@ -0,0 +1,29 @@
+require "dish"
+
+class Takeaway
+  attr_reader :cust_order, :dishes
+
+  def initialize
+    @dishes = [
+      Dish.new("Pizza", 5.20),
+      Dish.new("Kebab", 3.00),
+      Dish.new("Chips", 1.00),
+      Dish.new("Drink", 0.80),
+    ]
+  end
+
+  def view_menu
+    dishes.each do |dish|
+      puts "#{dish.name}" + " £" + ('%.2f' % dish.price).to_s
```

Comment:
Perhaps the formatting of the dish and price could be abstracted into its own method?